pygeoapi/plugin.py

```diff
@@ -60,6 +60,7 @@
         'SensorThings': 'pygeoapi.provider.sensorthings.SensorThingsProvider',
         'SQLiteGPKG': 'pygeoapi.provider.sqlite.SQLiteGPKGProvider',
         'Socrata': 'pygeoapi.provider.socrata.SODAServiceProvider',
+        'TinyDB': 'pygeoapi.provider.tinydb_.TinyDBProvider',
```

Comment:
Looks like `TinyDBCatalogueProvider` has been renamed to `TinyDBProvider` ? So can be removed here or for forward compat map to `TinyDBProvider`...

Reply:
(ok I see that both are defined).

Reply:
Yes, there are nuances between a feature and record provider here.